src/github.py

```diff
@@ -0,0 +1,62 @@
+import os
+import requests
+import streamlit as st
+
+ORG = 'NYCPlanning'
+PERSONAL_TOKEN = os.environ['GHP_TOKEN']
+headers = {'Authorization': 'Bearer %s' % PERSONAL_TOKEN}
+params = {'filter': 'all', 'per_page': 100}
+BASE_URL=f'https://api.github.com/repos/{ORG}'
+
+def parse_workflow(workflow):
+    return {
+        "status": workflow['status'],
+        'conclusion': workflow['conclusion'],
+        'timestamp': workflow['updated_at'],
+    }
+
+def get_workflow(repo, name): 
+    url = f"{BASE_URL}/{repo}/actions/workflows/{name}"
+    r = requests.get(url, headers=headers)
+    return r.json()
+
+def __get_workflow_runs_helper(url, params=None):
+    response = requests.get(url, headers=headers, params=params).json()
+    if 'workflow_runs' in response:
+        return response['workflow_runs']
+    else:
+        raise Exception(f'Error retreiving workflow runs from Github. If error persists, contact Data Engineering. Github API response: {response}')
+
+def get_workflow_runs(repo, workflow_name = None, items_per_page = None, total_items = None):
```

Comment:
Are the links in the QAQC checks under the `status` column meant to work? I just get sent back to the DE QAQC page

Reply:
Oh good catch - I think I just forgot to implement that. Doing it now

Reply:
Deployed fix